kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/partitioner/AvroFieldValuePartitionerConfig.java

```diff
@@ -0,0 +1,133 @@
+package org.hypertrace.core.kafkastreams.framework.partitioner;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.Maps;
+import com.google.common.util.concurrent.AtomicDouble;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.Value;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor
+@Getter
```

Comment:
Please switch to Value and keep immutable.

Reply:
fixed